index.html

```diff
@@ -23,9 +23,26 @@ <h1>
       <nav></nav>
     </header>
     <main>
-      <section>
-        <h2>4 week <span>workout plan</span></h2>
+      <section class="mx-auto hero">
+        <img
+          class="hero__image mx-auto"
+          src="images/hero-section-woman.png"
+          alt=""
+        />
+        <h2 class="hero__title">
+          4 week<span class="hero__subtitle">workout plan</span>
+        </h2>
+        <p class="hero__text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas
+          fringilla, lacus eget consequat tincidunt, lectus ex bibendum libero,
+          eget iaculis quam arcu quis mauris. Donec ex neque, egestas in
+          faucibus. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+          Maecenas
+        </p>
+        <br />
+        <a class="button" href="link to Next Page">Try for free</a>
```

Comment:
I'd recommend a different  "placeholder text" for the `href` as this is not a valid location. You could go with something like `href="#"` or `href="/try-it-for-free"`

Reply:
But the classname is fine!